scalameta/dialects/shared/src/main/scala/scala/meta/dialects/package.scala

```diff
@@ -147,15 +147,12 @@ package object dialects {
   )
 
   implicit val Scala212 = Scala211.copy(
-    allowLiteralTypes = false, // Scheduled to be included in 2.12.2
```

Comment:
Or alternatively I get this PR in a bit later today (in PST, meaning late night CEST) as we just discussed with @olafurpg.